NEWS

```diff
@@ -1,3 +1,11 @@
+25-Nov-2020 Ignition 2.8.0
+
+  Features:
+
+    - Support .d directory for base configs
+    - Support unmasking systemd units
+    - Allow obtaining a transcript of merge operations
```

Comment:
We've only had the `Public Go API Changes` section once, for breaking changes in 2.0.0.  So I'd be inclined to just put this under `Changes`, but I could be convinced otherwise.

If we use `Changes`, how about `Add Go merge API producing a transcript of merge operations`?

Reply:
hmm... I don't have a strong opinion here. I think it makes sense to include that information under a `Changes` section.